persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/StargateQueryHandler.java

```diff
@@ -332,8 +332,9 @@ private void authorizeAuthenticationStatement(
         authorization.authorizeRoleManagement(
             authenticationSubject, role, grantee, scope, SourceAPI.CQL);
       } catch (io.stargate.auth.UnauthorizedException e) {
+        logger.debug("Unauthorized statement: " + scope, e);
         throw new UnauthorizedException(
-            String.format("Missing correct permission on role %s", role));
+            String.format("Missing correct permission on role %s: %s", role, e.getMessage()));
```

Comment:
Isn't there any IT/unit-test that validates the exception message? I don't see any test changes in the stargate.

Reply:
unit tests added